README.md

```diff
@@ -12,6 +13,7 @@ A comprehensive linting solution that sweeps your code clean. Fly through your c
 
 - [Installation](#installation)
 - [Usage](#usage)
+- [Using pnpm](#using-pnpm)
```

Comment:
Good explanation for installation with alternative instruments. But the readme file has the `Installation` section. I think that instructions for it should be described on first section - currently it seems inconsistent for npm/yarn/pnpm. 

For example you can use subheadings for `Installation` section

Reply:
on it @dipiash